driver/src/models/account_state.rs

```diff
@@ -9,6 +9,10 @@ impl AccountState {
     pub fn read_balance(&self, token_id: u16, account_id: Address) -> u128 {
         self.0.get(&(account_id, token_id)).cloned().unwrap_or(0)
     }
+
+    pub fn user_token_pairs(&self) -> impl Iterator<Item = (Address, u16)> + '_ {
+        self.0.iter().map(|(&pair, _)| pair)
+    }
```

Comment:
Is there a test for this? I find this method so abstract that I am unable to determine what is should be returning. Would really help if I had an "expectation" example.